### ISSUE TITLE
Failed to resolve import in adding a page

### ISSUE THREAD

**Issue by amirhosseinfaraji** [ng-doc/ng-doc#123]
### Is this a regression?

I don't know

### Description

I am trying to implement simple documentation with this library and I used this [tutorial ](https://medium.com/@askoropad/ngdoc-documentation-for-angular-projects-3f6ea8fc22b0) in medium and general documentation. Unfortunately, when I try to add a page I get this error: 
`Internal server error: Failed to resolve import "ng-doc/document-sample/guides/installation/module" from ".angular\vite-root\document-sample\main.js". Does the file exist?`

I don't know it's a bug or no but, but I couldn't find any solution in the documentation

Angular CLI: 17.0.8
Node: 20.9.0
Package Manager: npm 10.1.0
@ng-doc/app: "17.2.1"
@ng-doc/builder: "17.2.1"
@ng-doc/core: "17.2.1"
@ng-doc/ui-kit: "17.2.1"

### OS

Windows

### Browser

Chrome

### Node version

20.9.0

**Comment by skoropadas** [comment]
Hey, thanks for the ticket! A lot has changed since the time of that article, so I would recommend using the documentation first and foremost, as it gets updated after each change.

It's hard to say what the reason is. I've encountered something similar in applications where there was no `baseUrl: "."` in the `tsconfig.json`. Could you create a repository where I could reproduce this?

**Comment by amirhosseinfaraji** [comment]
@skoropadas Thank you for your answer. Honestly, I used documentation but I couldn't find my answer. so because of that, I used your article.
I create this [repo ](https://github.com/amirhosseinfaraji/ng-doc-sample) and I remove the ng-docs directory from .gitignore for more information. I didn't do any additional things. I just created a new app with angular/cli after that I added ng-doc with the ng add command after that made a directory for docs and generated an installation page with this command `ng g @ng-doc/builder:page "Installation"` in docs directory. after serving the app I got the error related to failing to load the installation module in ng-doc.routing

**Comment by skoropadas** [comment]
@amirhosseinfaraji Thank you, I've checked the project, and it seems to be an issue with the `baseUrl`. Just add `baseUrl: "."` to your `tsconfig.json` in the `compilerOptions` section, and everything should start working.

Previously, this property was added by default when creating an Angular project. Apparently, something has changed in version 17. In any case, I will fix this in future releases by replacing imports with relative paths.
